src/modules/75-ci/components/PipelineStudio/BuildInfraSpecifications/BuildInfraSpecifications.tsx

```diff
@@ -696,6 +853,44 @@ export default function BuildInfraSpecifications({ children }: React.PropsWithCh
       </>
     )
 
+  const renderVolumesList = ({
+    value,
+    stringKey
+  }: {
+    value: string | string[] | any
+    stringKey: keyof StringsMap
+  }): JSX.Element | undefined =>
+    value && (
+      <>
+        <Text font={{ variation: FontVariation.FORM_LABEL }} margin={{ bottom: 'xsmall' }}>
+          {getString(stringKey)}
+        </Text>
+        {typeof value === 'string' ? (
+          <Text color="black">{value}</Text>
+        ) : (
+          <ul className={css.plainList}>
+            {value.map((row: { [key: string]: any }[], idx: number) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232285)
```suggestion
            {value.map((row: { [key: string]: unknown }[], idx: number) => {
```